docs/docs/APIGuide/Module.md

```diff
@@ -147,3 +147,262 @@ val preductResult = model.predict(predictSet)
  preductResult = model.predict(predictSet)
 ```
 
+## Module Freeze
+To "freeze" a module means to exclude some layers of model from training.
```

Comment:
means the parameter(weight, bias) of the module won't be changed during the training.